osbenchmark/metrics.py

```diff
@@ -1995,12 +1998,15 @@ def op_metrics(op_item, key, single_value=False):
     def v(self, d, k, default=None):
         return d.get(k, default) if d else default
 
-    def add_op_metrics(self, task, operation, throughput, latency, service_time, processing_time, error_rate, duration, meta):
+    def add_op_metrics(self, task, operation, throughput, latency,
```

Comment:
This is ripe for refactoring into an object passed around called `Metrics`. For a future/separate PR.